driver/compenv.ml

```diff
@@ -458,17 +458,18 @@ let read_one_param ppf position name v =
 let read_OCAMLPARAM ppf position =
   try
     let s = Sys.getenv "OCAMLPARAM" in
-    let (before, after) =
-      try
-        parse_args s
-      with SyntaxError s ->
-        print_error ppf s;
-        [],[]
-    in
-    List.iter (fun (name, v) -> read_one_param ppf position name v)
-      (match position with
-         Before_args -> before
-       | Before_compile _ | Before_link -> after)
+    if String.length s <> 0 then
```

Comment:
Why not directly compare with the empty string?
```suggestion
    if s <> "" then
```

Reply:
Indeed. I'm just too used to working in code bases where `<>` has type `int -> int -> int`. Updated now.